### PR TITLE
Docs for VictoryCursorContainer

### PR DESCRIPTION
**Pull request by chrisbolin**
Docs for the newly introduced VictoryCursorContainer.

(Also fixes a broken link and a spelling mistake, because I couldn't help myself.)

### CODE REVIEW

File: src/screens/docs/components/victory-cursor-container/ecology.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# VictoryCursorContainer
+
+`VictoryCursorContainer` adds a cursor to a chart to inspect coordinates.
+The cursor can either be a 2-dimensional crosshair, or a 1-dimensional line.
+The cursor moves with the mouse (or on touch on mobile devices) along the visible domain of the chart.
+The cursor can also display a label for the active coordinates using the `cursorLabel` prop.
+
+`VictorySelectionContainer` may be used with any Victory component that works with an x-y coordinate
```
Comment (boygirl) [inline src/screens/docs/components/victory-cursor-container/ecology.md]: `VictoryCursorContainer`

File: src/screens/docs/components/victory-cursor-container/ecology.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+system, and should be added as the `containerComponent` of the top-level component.
+However, the component that uses it must be standalone
+(`standalone={true}`), which is the default for all Victory components.
```
Comment (boygirl) [inline src/screens/docs/components/victory-cursor-container/ecology.md]: Maybe this should be "default for all top-level VictoryComponents"

File: src/screens/docs/components/victory-cursor-container/ecology.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+Note that the cursor allows you to inspect the entire domain, not just the data points.
+If you would like to instead highlight only the data points, consider using [VictoryVoronoiContainer].
+
+```jsx
```
Comment (boygirl) [inline src/screens/docs/components/victory-cursor-container/ecology.md]: Since this is now an ecology document, you can make this an actual playground

**Comment by boygirl** [comment]
@chrisbolin approved pending minor changes

File: src/screens/docs/components/victory-cursor-container/ecology.md (hunk 1 above)
Comment (chrisbolin) [inline src/screens/docs/components/victory-cursor-container/ecology.md]: oh dammit. sorry :P

File: src/screens/docs/components/victory-cursor-container/ecology.md (hunk 3 above)
Comment (chrisbolin) [inline src/screens/docs/components/victory-cursor-container/ecology.md]: oh nice! sounds good. how about if i save the playground for my next task, which is add playgrounds to all the components - does that work?

**Pull request merged**